wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1018,30 +1019,68 @@ instance BuildableSafeGen PasswordUpdate where
         pwdOld
         pwdNew
 
+data Accuracy
+    = Accurate
+    | LowerBound
+    deriving (Eq, Show, Generic)
+
+instance Arbitrary Accuracy where
+    arbitrary = elements [Accurate, LowerBound]
+
+instance ToSchema Accuracy where
+    declareNamedSchema _ = do
+        pure $ NamedSchema (Just "Accuracy") $ mempty
+            & type_ .~ SwaggerString
+            & enum_ ?~ ["Accurate", "LowerBound"]
+
+deriveSafeBuildable ''Accuracy
+instance BuildableSafeGen Accuracy where
+    buildSafeGen _ = show
+
+instance ToJSON Accuracy where
+    toJSON = String . show
+
+instance FromJSON Accuracy where
+    parseJSON = withText "Accuracy" $ \str ->
+        case str of
+            "Accurate" -> pure Accurate
+            "LowerBound" -> pure LowerBound
```

Comment:
I might be wrong, but I'm decently sure the style we picked was camel case, but _non-capitalised_, for enumerations. If I'm right, we should strive to keep everything uniform and maybe for the future even write a Swagger test to check that 😀